src/plugins/intel_gpu/src/kernel_selector/cl_kernels/fully_connected_gpu_bfyx_ref.cl

```diff
@@ -67,19 +72,24 @@ KERNEL(fc)(
            for (uint x = 0; x < INPUT0_SIZE_X; ++x)
             {
                 const uint input0_idx = INPUT0_GET_INDEX(b, ifm, y, x);
-                const uint filter_idx = GET_FILTER_INDEX(FILTER, 0, ofm, ifm, y, x);
                 #if COMPRESSED_WEIGHTS
-                    FILTER_TYPE filter_compressed = weights[filter_idx];
                     #if DECOMPRESSION_ZP_TERM
-                        ACCUMULATOR_TYPE zp = decompression_zp[DECOMPRESSION_ZP_GET_INDEX_SAFE(0, ofm, 0, 0)];
+                        ACCUMULATOR_TYPE zp = TO_ACCUMULATOR_TYPE(decompression_zp[DECOMPRESSION_ZP_GET_INDEX_SAFE(ofm, ifm / DECOMPRESSION_SCALE_GROUP_SIZE, 0, 0)]);
```

Comment:
Shouldn't we use `DECOMPRESSION_ZP_GROUP_SIZE` here and in other zero points-related calculations?

Reply:
Fixed in https://github.com/openvinotoolkit/openvino/pull/20491/commits/88932f73cd69f052f0e8937d8e898c253a9e2bac